osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderTick.cs

```diff
@@ -50,10 +52,8 @@ protected override void CheckForJudgements(bool userTriggered, double timeOffset
 
         protected override void UpdatePreemptState()
         {
-            this.Animate(
-                d => d.FadeIn(ANIM_DURATION),
-                d => d.ScaleTo(0.5f).ScaleTo(1f, ANIM_DURATION * 4, Easing.OutElasticHalf)
-            );
+            this.FadeOut().FadeIn(ANIM_DURATION);

```

Comment:
I'm confused about the fade out. If we start at Alpha 0 and never faded in isn't this unnecessary?

Reply:
i'll remove the alpha=0